src/main/groovy/com/bmuschko/gradle/docker/tasks/image/DockerBuildImage.groovy

```diff
@@ -111,6 +115,10 @@ class DockerBuildImage extends AbstractDockerRemoteApiTask implements RegistryCr
             buildImageCmd.withPull(getPull())
         }
 
+        if(getLabels()) {
```

Comment:
Coding style right now is to have a space after `if` and the opening brace. Sorry for being anal.